<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Plantas_medicinales</class>
 <widget class="QDialog" name="Frm_Plantas_medicinales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nombre Esp Esp:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtNombre_esp_esp"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Nombre Esp Mex:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="txtNombre_esp_mex"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblnombrecatala">
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Nombre  Català:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtNombre_cat"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Nombre Cient. Lat:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="txtNombre_lat"/>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="2" column="1">
       <widget class="QPushButton" name="btnSalir_2">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAnadir_2">
        <property name="text">
         <string>Añadir</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDeshacer_2">
        <property name="text">
         <string>Deshacer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnEditar_2">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnBorrar_2">
        <property name="text">
         <string>Baja</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnGuardar_2">
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Mostrar:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtFiltrar">
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff5500;&quot;&gt;GESTIÓN DE LA BD DE PLANTAS MEDICINALES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableView" name="tabla_plantas">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);
alternate-background-color: rgb(237, 255, 196);</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lblImagen_2">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Maya_Meditec.qrc">:/maya/PNG/PNG/Hierbas_medicinales.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Principios Activos:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtEstado"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Posología/
Preparación:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QTextEdit" name="txtPosologia"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Efectos-sec/
Contraindicaciones:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Estado:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QTextEdit" name="txtprincipios_activo"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QTextEdit" name="txtAccion"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Indicaciones/
Acción:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QTextEdit" name="txtEfectos"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>9</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Maya_Meditec.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSalir_2</sender>
   <signal>clicked()</signal>
   <receiver>Frm_Plantas_medicinales</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1036</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>1060</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
